src/main/webapp/WEB-INF/views/InvestmentPage.html

```diff
@@ -0,0 +1,317 @@
+		<div class="item">
+            <div class="thumbnail">
+                <div>
+                    <a href="#"><h3>Benefits</h3></a>
+					<hr>
+                </div>
+                <div class="caption">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                        labore
+                        et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
+                        ut
+                        aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+                        cillum dolore eu fugiat nulla pariatur.</p>
+                    <hr>
+                </div>
+            </div>
+        </div>
+		<div class="item">
+            <div class="thumbnail">
+                <div>
+                    <a href="#"><h3>Bussiness model</h3></a>
+					<hr>
+                </div>
+                <div class="caption">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                        labore
+                        et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
+                        ut
+                        aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+                        cillum dolore eu fugiat nulla pariatur.</p>
+                    <hr>
+                </div>
+            </div>
+        </div>
+		<div class="item">
+            <div class="thumbnail">
+                <div>
+                    <a href="#"><h3>Risks</h3></a>
+					<hr>
+                </div>
+                <div class="caption">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                        labore
+                        et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
+                        ut
+                        aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+                        cillum dolore eu fugiat nulla pariatur.</p>
+                    <hr>
+                </div>
+            </div>
+        </div>
+    </div>
+	<div data-columns>
+	    <div class="item">
+            <div class="thumbnail">
+                <div class="caption centered">
+                    <a href="#"><h3>Min investment $10000</h3></a>
+                </div>
+                <button type="button" class="btn btn-success" data-toggle="modal" data-target="#modal-1">I want to invest</button>
+				<!--Modal-->
+				<div class="modal fade" id="modal-1">
+					<div class="modal-dialog">
+						<div class="modal-content ">
+							<div class="modal-header">
+								<button class="close" type="button" data-dismiss="modal">
+									<i class="fa fa-close"></i>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)